source/includes/concept/_dataset.md

```diff
@@ -0,0 +1,76 @@
+# Concepts
+
+## Dataset
+
+Resource Watch API's goal is to provide a common interface for interacting with different data sources. A **Dataset** is a resource that represents how to access these data sources, and it is the cornerstone onto which many other API resources build upon.
```

Comment:
I see and agree with your point, but I don't know if I would emphasize so early on in the docs that we have special good stuff for geospatial data. To clarify: my goal here is to give someone who's never heard of our API a quick teaser/description of what the API can do for them. And while we can do extra cool stuff for people with geospatial data, I also don't want to transmit the idea that, if their data is not rgeoeferenced, they become sort of second class citizens.

Reply:
I added a few more examples below where I cover different use cases for georeferenced data. Do you think this achieves the goal you had in mind, or shall we rephrase this paragraph?

Reply:
The examples you give in the later revision help with this. Definitely a drawback of this Github review process - I actually can't see the most recent revisions in the context of this conversation, because the conversation is tied to an earlier set of edits.